ClientApp/src/components/admin/overview/OverviewMacro.tsx

```diff
@@ -78,14 +83,28 @@ const OverviewMacro: React.FC<IOverviewMacro> = ({ accessToken, location }) => {
         setSelectedConnection((prevState) => {
           return {
             ...prevState,
-            recipient: undefined,
+            recipient: {} as RecipientType,
           };
         });
       } else {
         setSelectedConnection((prevState) => {
           return {
             ...prevState,
             recipient: newRecipient,
+            editRecipient: {
+              rowKey: newRecipient.rowKey,
+              partitionKey: newRecipient.partitionKey,
+              familyId: newRecipient.familyId.toString(),
+              dinner: newRecipient.dinner,
+              dessert: newRecipient.dessert,
+              note: newRecipient.note,
+              contactFullName: newRecipient.contactFullName,
+              contactEmail: newRecipient.contactEmail,
+              contactPhoneNumber: newRecipient.contactPhoneNumber,
+              institution: newRecipient.institution,
+              referenceId: newRecipient.referenceId,
+              familyMembers: newRecipient.familyMembers as Types.PersonType[],
+            } as Types.RecipientType
```

Comment:
Maybe this function could easily map the properties?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

The usage of two classes with the same name is confusing. Why are they seperated and should they be seperate? 

Might be an idea to remove usage of one in favour of the other, or create a subclass

Reply:
I were no sure if that function were working due to the recipient objects originates from different types, but in fact it does!

I got it working writing it like this:
```cs
Object.assign({} as Types.RecipientType, recipient);
```